plugins/view-resources/src/components/navigator/TreeElement.svelte

```diff
@@ -91,6 +104,19 @@
   <div class="an-element__grow" />
 
   {#if !parent}
+    {#each inlineActions as action}
+      <div
+        class="an-element__tool"
+        class:pressed={hovered}
+        on:click|preventDefault|stopPropagation={(ev) => onInlineClick(ev, action)}
+      >
+        {#if action.icon !== undefined}
```

Comment:
nit: can be simplified to one line:
`<Icon icon={action.icon ?? ActionIcon} size={'small'} />`